test/functional/feature_pruning.py

```diff
@@ -10,25 +10,25 @@
 """
 import os
 
-from test_framework.blocktools import create_coinbase
-from test_framework.messages import CBlock
-from test_framework.script import (
-    CScript,
-    OP_NOP,
-    OP_RETURN,
-)
+from test_framework.blocktools import create_large_block
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import (
     assert_equal,
     assert_greater_than,
     assert_raises_rpc_error,
 )
+from test_framework.script import (
+    CScript,
+    OP_RETURN,
+    OP_TRUE,
+)
 
 # Rescans start at the earliest block up to 2 hours before a key timestamp, so
 # the manual prune RPC avoids pruning blocks in the same window to be
 # compatible with pruning based on key creation time.
 TIMESTAMP_WINDOW = 2 * 60 * 60
 
+scriptPubKey = CScript([OP_RETURN] + [OP_TRUE] * 950000)
```

Comment:
In commit "rpc: Enable wallet import on pruned nodes and add test" (f42c077a8f948f5c93b68eb1b7fa246b4f6475ef)

Just IMO, but I think it would be better to keep the existing `big_script` variable below and delete this new variable (smaller diff, more descriptive variable name, more straightforward code avoiding global variables)